modes/magit/evil-collection-magit.el

```diff
@@ -1,36 +1,52 @@
-;;; evil-collection-magit.el --- Bindings for `magit' -*- lexical-binding: t -*-
+;;; evil-collection-magit.el --- evil-based key bindings for magit
 
-;; Copyright (C) 2018 James Nguyen
+;; Copyright (C) 2015-2016 Justin Burkett
 
-;; Author: James Nguyen <james@jojojames.com>
-;; Maintainer: James Nguyen <james@jojojames.com>
+;; Author: Justin Burkett <justin@burkett.cc>
+;; Maintainer: Justin Burkett <justin@burkett.cc>
+;; James Nguyen <james@jojojames.com>
 ;; Pierre Neidhardt <mail@ambrevar.xyz>
-;; URL: https://github.com/emacs-evil/evil-collection
-;; Version: 0.0.1
-;; Package-Requires: ((emacs "25.1"))
-;; Keywords: evil, emacs, tools
+;; Package-Requires: ((evil "1.2.3") (magit "2.6.0"))
+;; Homepage: https://github.com/emacs-evil/evil-collection
+;; Version: 0.4.1
 
-;; This program is free software; you can redistribute it and/or modify
-;; it under the terms of the GNU General Public License as published by
-;; the Free Software Foundation, either version 3 of the License, or
-;; (at your option) any later version.
-
-;; This program is distributed in the hope that it will be useful,
+;; This file is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published
+;; by the Free Software Foundation; either version 3, or (at your
+;; option) any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
 ;; but WITHOUT ANY WARRANTY; without even the implied warranty of
 ;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ;; GNU General Public License for more details.
-
-;; You should have received a copy of the GNU General Public License
-;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+;;
+;; For a full copy of the GNU General Public License
+;; see <http://www.gnu.org/licenses/>.
 
 ;;; Commentary:
-;;; Bindings for `magit'
-;;; This file is to work around an issue described in
-;;; https://github.com/emacs-evil/evil-collection/issues/108
-;;; Ideally this file is only temporary and should be removed once
-;;; #108 is resolved.
+
+;; This library configures Magit and Evil to play well with each
+;; other. For some background see https://github.com/magit/evil-magit/issues/1.
+
+;; Installation and Use
```

Comment:
This could probably use some attention, I removed some stuff but the doc is not totally appropriate now that it's in evil-collection.

Reply:
I would just keep the information in the readme. It's not very accessible here anyway

Reply:
Sounds good, removed.